bridge/opentracing/bridge.go

```diff
@@ -503,19 +520,19 @@ func otTagToOTelLabel(k string, v interface{}) label.KeyValue {
 	case int64:
 		return key.Int64(val)
 	case uint64:
-		return key.Uint64(val)
+		return key.Int64(int64(val))
```

Comment:
Casting from uint64 to int64 isn't safe.  Do we need to fall back to string here?